app/assets/stylesheets/spotlight/_collapse_toggle.scss

```diff
@@ -0,0 +1,18 @@
+.btn.collapse-toggle {
+  // adjust as needed, taken from bootstrap.css glyphicons
+  &:after {
+    // symbol for "opening" panels
+    font-family: 'Glyphicons Halflings';
```

Comment:
Properties should be ordered color, content, float, font-family, font-size, line-height, padding-left<br>Prefer double-quoted strings